NuCivic/dkan
# Geofield map button toggling is not working properly [CIVIC-4774]

Issue: CIVIC-4774

## Description
Improvement for the Geofield widget.
This PR include only added/altered behat tests. The main [PR](https://github.com/NuCivic/leaflet_draw_widget/pull/10) is listed on the PR dependencies. 

## Acceptance Criteria
1. Different Geofield input fields should be exclusively visible.
2. Toggle buttons should be aware of the status of other toggle buttons.

## QA Tests
1. Go to the dataset edit page.
2. Use the "Point", "GeoJson", and "Use map" button to switch the Geofield input fields.

This will lead to an extra irremovable text field to appear. Multiple buttons with the same label, and multiple input field to show up at the same time not respecting the exclusivity.

Using the PR on the leaflet_draw_widget should provide improved UI.

## PR dependencies
- https://github.com/NuCivic/leaflet_draw_widget/pull/10

## Merging instructions:
- [x] https://github.com/NuCivic/leaflet_draw_widget/pull/10 should be merged first.
- [x] Discard or revert the changes to done to the drupal-org.make file before merging this PR.
